net/libreswan/patches/051-nftables.patch

```diff
@@ -0,0 +1,137 @@
+--- a/programs/ipsec/ipsec.in
```

Comment:
What do we need this patch for? I haven't looked at it closely, but I thought the firewall rules gets set via the hotplug scripts iptables-hotplug/nftables-hotplug

Reply:
This patch is backported from upstream for nftables support. It's used in ```ipsec checknflog```

Reply:
This patch partially impements nflog so removed,